code/modules/antagonists/ert/ert.dm

```diff
@@ -236,6 +236,8 @@
 	if(isplasmaman(H))
 		H.equipOutfit(plasmaman_outfit)
 		H.open_internals(H.get_item_for_held_index(2))
+	else
+		H.dna.species.give_important_for_life(H)
```

Comment:
Plasmamen should use the same system, and any other species with required internals probably also wants those internals to default to on. You should by default call give_important_for_life() and then check if internals are equiped, if so enable them.

Reply:
give_important_for_life enables the internals itself. Plasmaman ert has it's own special outfit that give_important_for_life conflicts with.

Reply:
I don't see anything enabling internals in give_important_for_life. Regardless, what I said stands and plasmamen shouldn't need special handling here.

Reply:
@ninjanomnom outfit.dm:274 enables internals. Tested latest commit and it's working.

Reply:
![image](https://github.com/user-attachments/assets/62aec7da-6031-4d1f-bd7f-3cee1c6d04a5)

Reply:
Ah I see, it's in equip. I wonder if that's new since the old code that used to equip then open internals here.